CHANGELOG.md

```diff
@@ -4,3 +4,4 @@
 - Fixes issue where `emulators:exec` could fail to shut down cleanly (#2477).
 - Fixes issue where database emulator did not properly load initial rules (#2483).
 - Allow starting the UI with `emulators:exec` using the `--ui` flag.
+- `firebase init functions` templates now use Firebase Admin SDK v9.
```

Comment:
This is that weird catch-22 feeling where the CLI will support Node 8 but functions created by the CLI in Node 8 could fail because a dependency requires Node 10. I think if we're going to do this, we're going to have to make a breaking CLI change (which I'm not opposed to, but we should start collecting things we need to add to that).